app/assets/javascripts/float-kit/package.json

```diff
@@ -25,6 +25,27 @@
     "@ember/string": "^3.1.1",
     "@embroider/test-setup": "^3.0.2",
     "@glimmer/component": "^1.1.2",
+    "@types/ember": "^4.0.6",
```

Comment:
Is there no way to have shared package define all of this?

Reply:
I tried! But unfortunately not. Even though `@types/ember` lists all the others as transitive dependencies, only direct dependencies can be imported by an app. And so the same logic applies to these types packages.

Good news it that we'll be able to drop them once we get to Ember 5 - types are shipped in the main `ember-source` package